assets/css/_info-bubble.scss

```diff
@@ -248,6 +246,43 @@
         margin-bottom: 2px;
       }
     }
+
+    button {
+      margin-left: 3px;
+      margin-top: 0;
+      white-space: nowrap;
+
+      position: relative;
+
+      .icon {
+        opacity: 0.666;
+      }
+
+      &:active {
+        .icon {
+          margin-top: 1px;
+          opacity: 0.9;
+        }
+      }
+
+      &[disabled] {
+        background: darken($ui-colour, 10%);
+        color: black;
+        opacity: 1;
+        border-top-color: darken($ui-colour, 30%);
+        border-bottom-color: darken($ui-colour, 30%);
+
+        .icon {
+          opacity: 1;
+        }
+      }
+    }
+  }
```

Comment:
Is this block of CSS duplicating a similar block nested under `.variants` (line 102 of `_info-bubble.scss`)? If so, we should figure out how to refactor this to limit the amount of nesting that is needed & also not duplicate the CSS.